src/app/mood-ometer/mood-ometer.component.ts

```diff
@@ -1,15 +1,25 @@
 import { Component, OnInit } from '@angular/core';
+import {Observable} from 'rxjs';
+import {catchError} from 'rxjs/operators';
+import { HttpClient } from '@angular/common/http';
 
 @Component({
   selector: 'app-mood-ometer',
   templateUrl: './mood-ometer.component.html',
   styleUrls: ['./mood-ometer.component.css']
 })
 export class MoodOMeterComponent implements OnInit {
-userChoice: string;
+  constructor(private http: HttpClient) {}
+urlMoodSelectPost = 'http://10.3.4.111:4200/alorwebapp/rest/MoodMeter/create';
```

Comment:
try to use here the environments variables and fetch that data from a centralized place